ts/components/ContextualInfo.tsx

```diff
@@ -54,7 +54,7 @@ export default class ContextualInfo extends React.Component<Props> {
           }}
         />
         <ContentWrapper>
-          <H1>{this.props.title}</H1>
+          <H2>{this.props.title}</H2>
```

Comment:
Is the change from `H1` to `H2` intentional, or was it an oversight?

Reply:
Since I've seen similar scenarios in other files, I believe this is an intentional change

Reply:
Thank you!

Reply:
@ChrisMattew Yes, it's normal. From the **How to test** section in the PR description:
> Keep in mind that some slight changes are normal because there isn't a perfect match between legacy and new typographic styles.

The changes have been made to reduce discrepancies as much as possible, especially in size, compared to the previous styles.